test/dlc_tests/container_tests/bin/testTensorBoard

```diff
@@ -2,7 +2,7 @@
   
 HOME_DIR=/test
 BIN_DIR=${HOME_DIR}/bin
-LOG_DIR=/logs/train/plugins/profile
+LOG_DIR=/logs/plugins/profile
```

Comment:
I ran the test locally and there is no plugins dir in "train" - we need to get to the bottom of why, though, before accepting this change

Reply:
Additionally, if this is in fact behavior change from TF2.6 to TF2.7, we need to handle this in a way that's backward compatible

Reply:
Lets create a github issue on this meanwhile?

Reply:
@hsl89 do you have an understanding of why this is happening? I know there are some changes in keras behavior, but what are the effects on the customer experience when using tensorboard profiler?